pkgs/top-level/all-packages.nix

```diff
@@ -15733,7 +15732,7 @@ in
   mariadb = callPackage ../servers/sql/mariadb {
     # As per mariadb's cmake, "static jemalloc_pic.a can only be used up to jemalloc 4".
     # https://jira.mariadb.org/browse/MDEV-15034
```

Comment:
I couldn't find that reference in `MDEV-15034`. It seems more like mariadb simply didn't adapt their code to jemalloc 5:
https://jira.mariadb.org/browse/MDEV-15034?focusedCommentId=106074&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-106074

If this is still broken, we should at least create an issue upstream so it'll ever get fixed.

Also, the [upstream percona issue](https://jira.percona.com/browse/PS-4393) is closed and mentions percona should work with jemalloc 5 just fine.

Reply:
Jemmaloc 5 worked on MariaDB 10.5.1. With 10.4 and 10.3 error builld

Reply:
Mariadb still is on 10.3 in nixpkgs master.

What about we just update the version in nixpkgs to the latest release?

Reply:
See https://github.com/NixOS/nixpkgs/pull/74486.

Reply:
@flokli MariaDB 10.4 not worked with jammaloc 5. Need wait for release 10.5?
Test Mariadb 10.5.1 here - https://github.com/Izorkin/nur-packages/blob/master/default.nix#L12
Builded with jammalloc 5 and pcre2.